cmd/admin-revoker/main.go

```diff
@@ -235,6 +229,28 @@ func main() {
 				}
 			},
 		},
+		{
+			Name:  "auth-revoke",
+			Usage: "Revoke all pending/valid authorizations for a domain",
+			Action: func(c *cli.Context) {
+				domain := c.Args().First()
+				_, _, _, sac, stats := setupContext(c)
+				auths, err := sac.GetAuthorizationsByDomain(core.AcmeIdentifier{Type: core.IdentifierDNS, Value: domain})
+				if err != nil {
+					cmd.FailOnError(err, "Failed to retrieve authorizations")
+				}
+				for _, a := range auths {
```

Comment:
Before starting this loop, would be good to print how many auths are going to be revoked.